### PR TITLE
Add Bourgeau 75% PCB.

### PR DESCRIPTION
**Pull request by studiokestra**
## Description

Add QMK & VIA support for bourgeau 75% PCB, meant to be used in the PLA75 keyboard.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/studiokestra/bourgeau/info.json [hunk 1]
```diff
@@ -0,0 +1,96 @@
+{
+    "keyboard_name": "Bourgeau",
+    "url": "https://studiokestra.ca/bourgeau/",
+    "maintainer": "Studio Kestra",
+    "width": 16, 
+    "height": 6,
```
Comment (drashna) [inline keyboards/studiokestra/bourgeau/info.json:6]:
```suggestion
```

File: keyboards/studiokestra/bourgeau/rules.mk [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
```
Comment (drashna) [inline keyboards/studiokestra/bourgeau/rules.mk:10]: There is no "full" bootmagic anymore, only the "lite" option, which can be turned on with just "yes".
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```

File: keyboards/studiokestra/bourgeau/rules.mk [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
```
Comment (drashna) [inline keyboards/studiokestra/bourgeau/rules.mk:21]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by studiokestra** [comment]
Changes made as requested. Thanks @drashna !

File: keyboards/studiokestra/bourgeau/config.h [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+// /*== or choose animations ==*/
+//   #define RGBLIGHT_EFFECT_BREATHING
+//   #define RGBLIGHT_EFFECT_RAINBOW_MOOD
+//   #define RGBLIGHT_EFFECT_RAINBOW_SWIRL
+//   #define RGBLIGHT_EFFECT_SNAKE
+//   #define RGBLIGHT_EFFECT_KNIGHT
+//   #define RGBLIGHT_EFFECT_CHRISTMAS
+//   #define RGBLIGHT_EFFECT_STATIC_GRADIENT
+//   #define RGBLIGHT_EFFECT_RGB_TEST
+//   #define RGBLIGHT_EFFECT_ALTERNATING
```
Comment (drashna) [inline keyboards/studiokestra/bourgeau/config.h:58]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by studiokestra** [comment]
I haven't explicitly defined RGB definitions in the past. 
Is this part of the standard review moving forward or is it more of a preference thing?

**Comment by drashna** [comment]
> I haven't explicitly defined RGB definitions in the past. Is this part of the standard review moving forward or is it more of a preference thing?

It's a newer thing, since we've had a number of animations added, and have had issues with a lot of boards compiling over size.  By explicitly defining them means that this won't be a problem in the future. 

(also, the define for all animations wasn't originally meant to enable all the modes)

**Comment by studiokestra** [comment]
> > I haven't explicitly defined RGB definitions in the past. Is this part of the standard review moving forward or is it more of a preference thing?
> 
> It's a newer thing, since we've had a number of animations added, and have had issues with a lot of boards compiling over size. By explicitly defining them means that this won't be a problem in the future.
> 
> (also, the define for all animations wasn't originally meant to enable all the modes)

Makes sense, thanks for clarifying. Changes have been committed.

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**